xml/selinux.xml

```diff
@@ -450,54 +388,82 @@ setroubleshoot-server
 
 File contexts:
-Controlling term:               root:object_r:user_devpts_t
-/etc/passwd                     system_u:object_r:etc_t
-/etc/shadow                     system_u:object_r:shadow_t
-/bin/bash                       system_u:object_r:shell_exec_t
-/bin/login                      system_u:object_r:login_exec_t
-/bin/sh                         system_u:object_r:bin_t -&gt; system_u:object_r:shell_exec_t
-/sbin/agetty                    system_u:object_r:getty_exec_t
-/sbin/init                      system_u:object_r:init_exec_t
-/sbin/mingetty                  system_u:object_r:getty_exec_t
-/usr/sbin/sshd                  system_u:object_r:sshd_exec_t
-/lib/libc.so.6                  system_u:object_r:lib_t -&gt; system_u:object_r:lib_t
-/lib/ld-linux.so.2              system_u:object_r:lib_t -&gt; system_u:object_r:ld_so_t</screen>
+Controlling terminal:           unconfined_u:object_r:user_tty_device_t:s0
+/etc/passwd                     system_u:object_r:passwd_file_t:s0
+/etc/shadow                     system_u:object_r:shadow_t:s0
+/bin/bash                       system_u:object_r:shell_exec_t:s0 -> system_u:object_r:shell_exec_t:s0
+/bin/login                      system_u:object_r:login_exec_t:s0
+/bin/sh                         system_u:object_r:bin_t:s0 -> system_u:object_r:shell_exec_t:s0
+/sbin/agetty                    system_u:object_r:getty_exec_t:s0
+/sbin/init                      system_u:object_r:bin_t:s0 -> system_u:object_r:init_exec_t:s0
+/usr/sbin/sshd                  system_u:object_r:sshd_exec_t:s0</screen>
   </example>
  </sect1>
 
- <sect1 xml:id="sec-selinux-configure">
-  <title>Configuring &selnx;</title>
+ <sect1 xml:id="sec-selinux-mode-enforcing">
+  <title>Putting &selnx; into enforcing mode</title>
   <para>
    At this point you have a completely functional &selnx; system and it is
    time to further configure it. In the current status, &selnx; is
    operational but not in enforcing mode. This means that it does not limit
    any activities, and it logs everything that it should be doing if it
    were in enforcing mode. Review the log files to learn what activities
-   are not allowed. As a first
-   test, put &selnx; in enforcing mode and find out if you can still use
-   your server after doing so: check that the option
-   <option>enforcing=1</option> is set in the &grub; configuration file,
-   while <option>SELINUX=enforcing</option> is set in
-   <filename>/etc/selinux/config</filename>. Reboot your server and see if
-   it still comes up the way you expect it to. If it does, leave it like
-   that and start modifying the server in a way that everything works as
-   expected. However, you may not even be able to boot the server properly.
-   In that case, switch back to the mode where &selnx; is not enforcing and
-   start tuning your server.
+   are not allowed.
   </para>
+  <para>After running &selnx; in permissive mode first, you can try to
```

Comment:
I would add here an example on how to query the logs via e.g. `ausearch` or refer to the Troubleshooting section where this is described.

If we want to add such a example paragraph here, we could suggest

    > ausearch -ts boot

or `less /var/log/audit/audit.log` to remain consistent with the way described in the Troubleshooting section.

Reply:
Good point! I mentioned `less /var/log/audit/audit.log` now and pointed to the troubleshooting section for more details. I added this not at this exact point, but a few sentences later, after the following sentence: 
"In case you should not able to boot the server properly with SELinux in enforcing mode, switch back to permissive mode. Check the log files with less `/var/log/audit/audit.log`. For more details, see 'Troubleshooting'.